package.json

```diff
@@ -4,11 +4,16 @@
   "packageManager": "pnpm@9.12.3+sha512.cce0f9de9c5a7c95bef944169cc5dfe8741abfb145078c0d508b868056848a87c81e626246cb60967cbd7fd29a6c062ef73ff840d96b3c86c40ac92cf4a813ee",
   "scripts": {
     "build": "turbo build",
-    "test": "turbo test",
+    "test": "pnpm test:affected",
+    "test:affected": "turbo test --affected --output-logs=new-only",
+    "test:all": "turbo test",
+    "test:all:force": "turbo test --force",
+    "test:ci": "turbo test --concurrency=5",
+    "test:ci:affected": "pnpm test:ci --affected",
```

Comment:
If we make a really large PR, like let's say we update all the .gitignores in the whole project at once, then this will run into problems as well because it will run all the tests. I think we probably need to keep the --concurrency=5 here too.

Reply:
The flag is preserved in `test:ci:affected` since it calls `test:ci`